oeps/oep-0030-arch-pii-markup-and-auditing.rst

```diff
@@ -0,0 +1,269 @@
++---------------+------------------------------------------------------------+
+| Author        | Brian Mesick <bmesick@edx.org>                             |
++---------------+------------------------------------------------------------+
+| Arbiter       | TBD                                                        |
++---------------+------------------------------------------------------------+
+| Status        | Draft                                                      |
++---------------+------------------------------------------------------------+
+| Type          | Architecture                                               |
++---------------+------------------------------------------------------------+
+| Created       | 2018-08-29                                                 |
++---------------+------------------------------------------------------------+
+| Resolution    | `Original pull request`_                                   |
++---------------+------------------------------------------------------------+
+| References    | - `NIST Special Publication 800-122 (pdf)`_                |
++---------------+------------------------------------------------------------+
+
+.. _Original pull request: https://github.com/edx/open-edx-proposals/pull/
+.. _NIST Special Publication 800-122 (pdf): http://nvlpubs.nist.gov/nistpubs/Legacy/SP/nistspecialpublication800-122.pdf
+
+Abstract
+========
+
+The Open edX platform and related independently deployable applications (IDAs) must store information about learners that identifies the learner and, in some cases, is private to them. As an entity that stores this type of information we strive to exercise caution and respect for the privacy of the learners who use our services. This OEP defines what we consider to be personally identifying information (PII) and the ways in which we keep track of that information. It also introduces processes that will allow us to explicitly document which information stored in our systems is PII and provides mechanisms for helping developers follow the outlined best practices.
+
+This OEP explicitly does not cover:
+
+- The retirement process itself
+- The process for cataloging PII stored in operational logs such as Splunk or New Relic
+- The process for cataloging PII stored in non-operational systems such as analytics databases
+
+Motivation
+==========
+
+The primary goal of this OEP is to allow Open edX administrators and developers to understand which information needs special handling, and identifies our best practices for how to make those designations obvious. Respectful handling of PII depends on the following principles, which must be measured against our ability to run the service and meet our research goals and partner obligations:
+
+#. Storing the minimum personal data necessary
+#. Sharing the minimum personal data necessary
+#. Removing the maximum amount of stored personal information when requested
```

Comment:
How is #3 in this list different from #2?

Reply:
@nasthagiri The second point refers to the sharing of information with other entities, while the third is about retiring as much as possible.